machine-learning/app/models/base.py

```diff
@@ -139,11 +140,11 @@ def clear_cache(self) -> None:
 
 
 # HF deep copies configs, so we need to make session options picklable
-class PicklableSessionOptions(ort.SessionOptions):
+class PicklableSessionOptions(ort.SessionOptions):  # type: ignore[misc]
```

Comment:
I'd love to get rid of this comment, but the typing here is unfortunate. onnxruntime doesn't provide type stubs for this class so mypy defaults it to `Any`. Under strict typing, `Any` is not a valid base class to extend from.